paper-fab.js

```diff
@@ -154,11 +112,55 @@ $_documentContainer.innerHTML = `<dom-module id="paper-fab">
     <iron-icon id="icon" hidden\$="{{!_computeIsIconFab(icon, src)}}" src="[[src]]" icon="[[icon]]"></iron-icon>
     <span hidden\$="{{_computeIsIconFab(icon, src)}}">{{label}}</span>
   </template>
-
-  
 </dom-module>`;
+template.setAttribute('style', 'display: none;');
+document.body.appendChild(template.content);
+
+/**
+Material design: [Floating Action
+Button](https://www.google.com/design/spec/components/buttons-floating-action-button.html)
+
+`paper-fab` is a floating action button. It contains an image placed in the
+center and comes in two sizes: regular size and a smaller size by applying the
+attribute `mini`. When the user touches the button, a ripple effect emanates
+from the center of the button.
+
+You may import `iron-icons` to use with this element, or provide a URL to a
+custom icon. See `iron-iconset` for more information about how to use a custom
+icon set.
+
+Example:
+
+    <link href="path/to/iron-icons/iron-icons.html" rel="import">
+
+    <paper-fab icon="add"></paper-fab>
+    <paper-fab mini icon="favorite"></paper-fab>
+    <paper-fab src="star.png"></paper-fab>
+
 
-document.head.appendChild($_documentContainer.content);
+### Styling
+
+The following custom properties and mixins are available for styling:
+
+Custom property | Description | Default
+----------------|-------------|----------
+`--paper-fab-background` | The background color of the button | `--accent-color`
+`--paper-fab-keyboard-focus-background` | The background color of the button
+when focused | `--paper-pink-900`
+`--paper-fab-disabled-background` | The background color of the button when it's
```

Comment:
This table is broken by webmat. You might want to add the following config in a `formatconfig.json`